### PR TITLE
initial release of adapter

### PR DESCRIPTION
**Pull request by ms1752**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+args = _parse_args(sys.argv)
+
+print args
+
+this_device = LocalDeviceObject(
+    objectName="clearblade",
+    objectIdentifier=599,
+    maxApduLengthAccepted=1024,
+    segmentationSupported="segmentedBoth",
+    vendorIdentifier=15
```
Comment (rreinold) [inline main.py]: Can we move these numbers to constants with names that explain them? They seem magical to me

File: main.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+print "BACnet-Adapter v" + adapter_version
+
+
+def _parse_args(argv):
+    parser = argparse.ArgumentParser(description='BACnetAdapter')
+    parser.add_argument('--systemKey', required=True, help='The System Key of the ClearBlade platform the BACnet adapter will connect to.')
+    parser.add_argument('--systemSecret', required=True, help='The System Secret of the ClearBlade platform the BACnet adapter will connect to.')
+    parser.add_argument('--deviceName', required=True, help='The name of the device, defined within the devices table of the ClearBlade platform, representing the BACnet Adapter.')
+    parser.add_argument('--activeKey', required=True, help='The Active Key, defined within the devices table of the ClearBlade platform, corresponding to the BACnet Adapter.')
+    parser.add_argument('--ipAddress', required=True, help='The ip address of the device this adapter is running on')
```
Comment (rreinold) [inline main.py]: Can we use localhost?

File: main.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+print "BACnet-Adapter v" + adapter_version
+
+
+def _parse_args(argv):
+    parser = argparse.ArgumentParser(description='BACnetAdapter')
+    parser.add_argument('--systemKey', required=True, help='The System Key of the ClearBlade platform the BACnet adapter will connect to.')
+    parser.add_argument('--systemSecret', required=True, help='The System Secret of the ClearBlade platform the BACnet adapter will connect to.')
+    parser.add_argument('--deviceName', required=True, help='The name of the device, defined within the devices table of the ClearBlade platform, representing the BACnet Adapter.')
+    parser.add_argument('--activeKey', required=True, help='The Active Key, defined within the devices table of the ClearBlade platform, corresponding to the BACnet Adapter.')
```
Comment (rreinold) [inline main.py]: Can we give examples of these parameter values at the end of the line? // ex. A548A3544..

File: main.py (hunk 2 above)
Comment (ms1752) [reply main.py]: Nope, looks like it has to be an IP for bacpypes to work. Added some text that says that specifically

File: main.py (hunk 3 above)
Comment (ms1752) [reply main.py]: Good call, done

File: main.py (hunk 1 above)
Comment (ms1752) [reply main.py]: Done, I added comments that explains the details to the best of my ability (I'm not a BACnet expert)

**Pull request merged**